_docs/developer/vm_install_using_vagrant.md

```diff
@@ -398,15 +398,30 @@ instructions.
 
    Replace `<port>` with the port you want to expose externally on the machine that is running the VM, and expose the specified port on the machine if necessary.
 
-3. Start the vagrant machine with `vagrant up ssh`.
+3. Start the vagrant machine with the SSH argument set:
 
-4. Retrieve the private key for the vagrant machine, located at `<SUBMITTY GIT REPO>/.vagrant/machines/<VM>/VM BACKEND>/private_key`.
+   **Windows**
+   ```
+   set SSH=y
+   vagrant up
+   ```
+
+   **macOS/Linux**
+   ```
+   SSH=y vagrant up
+   ```
+
+4. Retrieve the private key for the vagrant machine, located at `<SUBMITTY GIT REPO>/.vagrant/machines/<VM>/<VM BACKEND>/private_key`.
 
    At the time of writing, `<VM>` is `ubuntu-18.04`, and `<VM BACKEND>` is `virtualbox`.
 
 5. Use SSH to connect from the remote device to the machine that is running the VM, and use SSH port forwarding to forward the necessary ports.
 
-   For most things, you will only need to forward the `site` port and the `websockets` port (1501, 8443), which can be done like so:
+   The username to sign in is `root` and the authentication method is with private key (using the private key specified in part 4). If you encounter authentication issues, try adding `vagrant` as the password in addition to the private key.
+
+   For most things, you will only need to forward the `site` port and the `websockets` port (1501, 8443).
```

Comment:
Maybe specify that you have to do local port forwarding? I was confused at first when I tried this step because my client had me choose between Local, Remote, and Dynamic port forwarding, and it wasn't obvious to me since I had no experience with ssh.